content/docs/source-git/work-with-source-git/pull-upstream-fixes.md

```diff
@@ -0,0 +1,95 @@
+---
+title: "Pull fixes from the upstream"
+date: 2021-01-26T23:59:59+01:00
+draft: false
+weight: 2
+---
+
+# Pull fixes from the upstream
+
+One of the common tasks of being a downstream maintainer is to pull fixes from
+the upstream codebase. This starts by creating a new bugzilla with an issue in
+the distribution which requires a code-change in the affected package. The
```

Comment:
Is this also required in Fedora? Maybe we should explicitly clarify who the intended audience is for these docs?

Reply:
Bugzilla is not required in Fedora to push an update. I dropped that info and simplified this paragraph.

I'm sorry but I don't understand where and how to write about the intended audience.